src/main/java/net/minestom/server/ServerConsts.java

```diff
@@ -0,0 +1,12 @@
+package net.minestom.server;
+
+public interface ServerConsts {
```

Comment:
Should be put back as statics on MinecraftServer, this is unnecessary and breaking